src/firefly/java/edu/caltech/ipac/firefly/resources/master-table.csv

```diff
@@ -0,0 +1,51 @@
+name,missionLabel (project),missionId,dataProductLabel,dataProductId,acronym,instrumentId,instrumentLabel,wavebandId,wavebandDesc,wavelength (in microns),wavelengthDesc,projectTypeKEy,projectTypeDesc,minSizeDeg,maxSizeDeg,helpUrl,tooltip,apiType,atlasTable,dataType
+JSON DM-12001,project,,subProject,surveyKey,,,,surveyKeyBand,title,,,,,minRangeDeg,maxRangeDeg,,,Service,surveyKeyAlt,
```

Comment:
those 2 lines can be removed. Also the code associate to it. See comment on the class.

Reply:
You looked at the wrong table.  The one you looked at is the one you made.  The table used is irsa-master-table.csv.  I can delete the old one before I merge to the dev.

Reply:
yes, please, remove it before merge.